dev/tests/utils/ckeditor5-dirs.js

```diff
@@ -141,6 +139,29 @@ describe( 'utils', () => {
 						repositoryPath: '/workspace/path/ckeditor5/node_modules/ckeditor5-core'
 					} );
 			} );
+
+			it( 'should return only ckeditor5 directories in development mode, including root directory', () => {
+				sandbox.stub( ckeditor5Dirs, 'getDirectories', () => sourceDirectories );
+				sandbox.stub( ckeditor5Dirs, 'getDependencies', () => dependencies );
+				sandbox.stub( git, 'parseRepositoryUrl' ).returns( repositoryInfo );
+				const includeRoot = true;
+
+				const directories = ckeditor5Dirs.getDevDirectories( workspacePath, packageJSONDependencies, ckeditor5Path, includeRoot );
+
+				expect( directories.length ).equal( 3 );
+				expect( directories[ 0 ] ).eql( {
```

Comment:
Please don't use `eql()`. I find it super confusing that it's something else than `equal()` (it's deep equal right?).